__tests__/wallet/api/swapService.test.ts

```diff
@@ -0,0 +1,79 @@
+
+import { decryptString, encryptString, hashPassword, create } from '../../../src/wallet/api/swapService'
+import config from '../../../src/config';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+
+const mockAxiosAdapter = new MockAdapter(axios);
+
+describe('hashing and encrypting', () => {
+  it('should correctly hash a password', () => {
+    const password = '123';
+    const hashedPassword = hashPassword(password);
+
+    expect(hashedPassword).toHaveLength(64);
+  })
+
+  it('should correctly encrypt and decrypt a string', () => {
+    const originalString = 'PartialTx|123123||';
+    const password = 'strongPassword';
+
+    const encryptedString = encryptString(originalString, password);
+    expect(encryptedString.length !== originalString.length).toBe(true);
+
+    const decryptedString = decryptString(encryptedString, password);
+    expect(decryptedString).toStrictEqual(originalString);
+  })
+})
+
+describe('base url configuration', () => {
+  it('should throw when no url parameter was offered', () => {
+      expect(() => config.getSwapServiceBaseUrl())
+        .toThrowError('You should either provide a network or call setSwapServiceBaseUrl before calling this.');
+  })
+
+  it('should return mainnet address when requested', () => {
+      expect(config.getSwapServiceBaseUrl('mainnet'))
+        .toStrictEqual('https://atomic-swap-service.hathor.network/')
+  })
+
+  it('should return testnet address when requested', () => {
+      expect(config.getSwapServiceBaseUrl('testnet'))
+        .toContain('testnet')
+  })
+
+  it('should throw when an invalid network is requested', () => {
+    // @ts-ignore
+    expect(() => config.getSwapServiceBaseUrl('invalid'))
+      .toThrowError('You should set it explicitly.');
```

Comment:
:speech_balloon: I believe the correct message here is the following:
```suggestion
      .toThrowError("Network invalid doesn't have a correspondent Atomic Swap Service url. You should set it explicitly.");
```